keras/layers/core/einsum_dense_test.py

```diff
@@ -273,6 +273,34 @@ def test_einsum_dense_basics(
         if expected_bias_shape is not None:
             self.assertEqual(layer.bias.shape, expected_bias_shape)
 
+    def test_einsum_dense_kernel_setter(self):
+        equation = "bc...,cde->bde..."
+        bias_axes = "de"
+        input_shape = (2, 1, 2)
+        output_shape = (3, 4)
+        layer = layers.EinsumDense(
+            equation, output_shape=output_shape, bias_axes=bias_axes
+        )
+        layer.build(input_shape)
+        kernel = layer.kernel
+        layer.kernel = kernel + 1.0
```

Comment:
This will set the kernel to a tensor (it won't be trainable anymore). Is that intended?